pkgpanda/build/test_util.py

```diff
@@ -0,0 +1,86 @@
+import pkgpanda.build
+
+
+def test_hash_files_in_folder(tmpdir):
+    hash_files_in_folder = pkgpanda.build.hash_files_in_folder
+
+    with tmpdir.as_cwd():
+        # Empty folder
```

Comment:
Nit: It's just my opinion, but generally I am a big fan of small independent tests instead of "scenario" tests with intermediate steps depending on each others. I would have probably split this one into at least 5 separate tests.